dna_analyse/DNA_Analyse.py

```diff
@@ -1,4 +1,4 @@
-gueltige_nukleotide = ['A', 'T', 'G', 'C']
+gueltige_nukleotide = ['A', 'T', 'G', 'C', 'a', 't', 'g', 'c']
```

Comment:
recht schlau! Das kann tatsächlich in vielen Fällen eine gute Lösung sein.
Weil unser Code weiter unten sowieso nochmal mit `str.upper()` arbeiten muss, könnte man auch gleich sagen, man arbeitet mit der in Großbuchstaben verwandelten Sequenz, wo das nötig ist, undlässt diese Liste kurz. Mehr oder weniger Geschmackssache